src/scripts/main.js

```diff
@@ -358,3 +358,55 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+
+const names = people.map((person) => person.name);
+
+const gender = people.map((person) => ({
+  ...person,
+  sex: person.sex === 'm' ? 'Male' : 'Female',
+}));
+
+const born = people.map((person) => person.born);
+const died = people.map((person) => person.died);
+
+const century = people.map((person) => Math.ceil(person.died / 100));
+
+const ages = people.map((person) => (person.died -= person.born));
```

Comment:
The calculation of ages is incorrect. The use of the decrement operator '-=' modifies the 'died' property of each person, which is not intended. You should use subtraction instead: 'person.died - person.born'.